spec/app/dumpers/test_response_dumper.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+ResponseDumper.define 'Test' do
+  dump 'index_mulitple_requests', status_codes: %i[ok no_content] do
+    get root_path
+    delete url_for(action: :destroy, controller: 'test')
```

Comment:
IMO, `controller` should come first. Can make a symbol.

```suggestion
    delete url_for(controller: :test, action: :destroy)
```